### PR TITLE
Details book

### PR DESCRIPTION
**Pull request by masutu98**
https://edu-redmine.framgia.vn/issues/7753

### CODE REVIEW

**Comment by masutu98** [comment]
ready

**Comment by masutu98** [comment]
ready

File: app/views/books/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          <div class="preview-pic tab-content">
+            <%= image_tag @book.image_url.to_s, class: "active tab-pane" %>
+          </div>
+        </div>
+        <div class="details col-md-6">
+          <h3 class="product-title"><%= @book.name %></h3>
+          <p><b><%= t "admin.books.type" %> </b><%= @book.category.name %></p>
+          <p><b><%= t "admin.books.pages" %> </b><%= @book.num_of_pages %></p>
+          <p><b><%= t "admin.books.writers" %> </b><%= @book.writers %></p>
+          <p><b><%= t "admin.books.publisher" %> </b><%= @book.publisher.name %></p>
```
Comment (dothidiemthao) [inline app/views/books/show.html.erb]: sử dụng delegate nhé

**Comment by masutu98** [comment]
ready

**Review by dothidiemthao — APPROVED** [review]

**Pull request merged**